src/XCCDF/xccdf_session.c

```diff
@@ -97,6 +97,7 @@ struct xccdf_session {
 	struct {
 		char *arf_file;				///< Path to ARF file to export
 		char *xccdf_file;			///< Path to XCCDF file to export
+		char *xccdf_stig_viewer_file;		///< Path to STIG Viewer XCCDF file to export
```

Comment:
does this ever get initialized? I think this is memory garbage in non debug builds. See xccdf_session_new_from_source

Reply:
ok nvm, the whole thing uses calloc so it's all NULL. something else must be causing the test fails then.